lib/core/managers/user_repository.dart

```diff
@@ -260,7 +260,7 @@ class UserRepository {
       _logger.d("$tag - getInfo: $fetchedInfo info fetched.");
 
       if (_info != fetchedInfo) {
-        _info ??= fetchedInfo;
+        _info = fetchedInfo;
```

Comment:
Niice fix I would have never guess hahaha! 😆 What does `??=` even means? Is it like only assign when not null?

Reply:
The debugger helped me a lot :laughing: 
So `a ?? b` will return `b` if `a` is null BUT `a ??= b` will assign `b` to `a` ONLY if `a` is null...that was the issue, as long as there was a cache the info will not update.